import-marc.sh

```diff
@@ -67,7 +67,7 @@ fi
 ##################################################
 if [ -z "$VUFIND_HOME" ]
 then
-  VUFIND_HOME="/usr/local/vufind"
+  VUFIND_HOME=$(pwd)
```

Comment:
Would it be preferable to use one of the solutions discussed at https://stackoverflow.com/questions/4774054/reliable-way-for-a-bash-script-to-get-the-full-path-to-itself to obtain the absolute path of the script itself and use that as VUFIND_HOME by default? Might be a little more reliable than `pwd` if the user is executing the script from outside of $VUFIND_HOME.

Reply:
Just read over the answers again. Looks like realpath needs to be installed on ubuntu 12.04, even though it's part of coreutils. Maybe one of the solutions that uses $BASH_SOURCE?